models/Product.js

```diff
@@ -0,0 +1,14 @@
+const mongoose = require('mongoose')
+const Schema = mongoose.Schema
+
+const Product = new Schema({
+    name: { type: String, required: true },
```

Comment:
thêm ở required: [ true, "Please provide name field" ], thêm maxLength: [ 50, "Name can not be more than 50 character" ]